_data/help/online.yml

```diff
@@ -1,8 +1,13 @@
 - name: Mahjong Soul
-  description: Riichi Mahjong with anime-like visuals
-  description_pl: Riichi Mahjong w oprawie graficznej w stylu anime
+  description: popular riichi mahjong client with anime-like visuals
+  description_pl: popularny klient riichi mahjonga w oprawie graficznej w stylu anime
   url: https://mahjongsoul.game.yo-star.com/
 
+- name: Riichi City
+  description: riichi mahjong client with anime-like visuals
+  description_pl: klient riichi mahjonga w oprawie graficznej w stylu anime
+  url: https://riichi.city/
```

Comment:
skoro cert jest wystawiony na mahjong-jp.com to chyba lepiej linkować tam :thinking: